### DISCUSSION TITLE
What's the difference between leveraged-authorization-uuid and inherited-uuid props?

### DISCUSSION THREAD

**Discussion by jasswalkjr** [usnistgov/OSCAL#916] [Q&A] [answered]
In the oscal_leveraging-example_ssp.json document included with v1.0.0-rc1, components of type leveraged-system have two props:

1. leveraged-authorization-uuid, and
2. inherited-uuid

What is the difference between the two props? The description included in the component is a bit ambiguous.
[oscal_leveraging-example_ssp.zip](https://github.com/usnistgov/OSCAL/files/6279375/oscal_leveraging-example_ssp.zip)

**Comment by david-waltermire** [comment, marked as answer]
In a SSP the `leveraged-authorization-uuid` references the `leveraged-authorization` entry in the SSP. The `inherited-uuid` references the component in the SSP that represents the leveraged system for which implementation details are inherited from.
